src/main/java/com/backbase/ct/bbfuel/configurator/PaymentsConfigurator.java

```diff
@@ -65,6 +68,8 @@ public void ingestPaymentOrders(String externalUserId) {
                     randomArrangement = getRandomFromList(sepaCtArrangements);
                 } else if (PAYMENT_TYPE_ACH_DEBIT.equals(paymentType)) {
                     randomArrangement = getRandomFromList(achDebitArrangements);
+                } else if (PAYMENT_TYPE_US_FOREIGN_WIRE.equals(paymentType)) {
+                    randomArrangement = getRandomFromList(usForeignWireArrangements);
                 } else {
                     randomArrangement = getRandomFromList(usDomesticWireArrangements);
```

Comment:
better add the `else if` for US domestic wire and let the `else` throw an `IllegalStateException`

Reply:
done